evap/staff/views.py

```diff
@@ -2263,6 +2264,17 @@ def faq_section(request, section_id):
     return render(request, "staff_faq_section.html", template_data)
 
 
+@manager_required
+def edit_infotexts(request):
+    student_form = InfotextForm()
+    contributor_form = InfotextForm()
+    return render(
+        request,
+        "staff_edit_infotexts.html",
+        dict(student_infotext_form=student_form, contributor_infotext_form=contributor_form),
+    )
```

Comment:
This doesn't (yet) save the changes that the user submitted.

This looks like work in progress, so I'll change the PR to be a draft for now so it's easier for us to keep an overview of the PRs. Please mark it as ready for review when we should review again.